### PR TITLE
Fake-News-Detection ML Project added

### LINKED ISSUE #3818: [FEAT]:  Fake News Detection
**Describe the project you want to add with tech stack**
I would like to add Fake News Detection using ML and DL.

**Expected behavior**
In this project I am using machine learning algorithm that is “Linear Regression Model”. Deep learning model that is “Long Short- Term Memory”, sigmoid function, etc.
Flow of the project: -

1. Data collection -> 2. Data Cleaning -> 3. Pre-processing -> 4. Vectorization -> 5. Tokenization ->
6. Training & testing the model -> 7. Classification(output).

The project will give output as Real and Fake news.

**Screenshots (optional)**
1.
image
image

2.
image
image

### PR DESCRIPTION
**Pull request by Parul1606**
# Title and Issue number 

Close #3818 

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by TusharKesarwani** [comment]
Resolve conflicts

**Comment by Parul1606** [comment]
> Resolve conflicts

Done 👍🏼

File: README.md [hunk 1]
```diff
@@ -815,25 +815,28 @@ Happy Coding! -->
...
   <td align="center"><a href="https://github.com/pranjay-poddar/Dev-Geeks/tree/main/Js-Games/Mastermind">Mastermind</a></td>
   <td align="center"> 268.</td>
-
   <td align="center"> <a href="https://github.com/pranjay-poddar/Dev-Geeks/tree/main/Python Projects/Forex Exchanger">Web-Image Filter</a></td>
   <td align="center"> 272.</td>
   <td align="center"> <a href="https://github.com/pranjay-poddar/Dev-Geeks/tree/main/Js-Games/Cubula">Cubula</a></td>
-
+ </tr>
+  
+ <tr align="Center">
```
Comment (TusharKesarwani) [inline README.md:823]: 'c' in the "center" should be in lowercase

Comment (Parul1606) [reply README.md:823]: image.png

It's correct

Comment (Parul1606) [reply README.md]: Done👍🏼

**Comment by Parul1606** [comment]
Please review this PR as soon as possible @TusharKesarwani

File: README.md [hunk 2]
```diff
@@ -844,9 +847,9 @@ Happy Coding! -->
 
   <td align="center"> 278.</td>
   <td align="center"> <a href="https://github.com/pranjay-poddar/Dev-Geeks/tree/main/cpp-projects/Vaccine Management System">Vaccine Management System</a></td>
-
-
-
+  
+  <td align="center"> 271.</td>
+  <td align="center" <a href="https://github.com/Parul1606/Dev-Geeks/tree/test/ML%20Projects/Fake-News-Detection">Fake-News-Detection</a></td>
```
Comment (TusharKesarwani) [inline README.md:852]: td tag is not closed

Comment (Parul1606) [reply README.md]: @TusharKesarwani done with the changes 👍🏼

**Pull request closed without merging**